### PR TITLE
Deploy azure support

### PR DESCRIPTION
**Pull request by ludop-usharesoft**
Add Azure Resource Manager support into deployment documentation.

### CODE REVIEW

File: end-user/en/source/pages/templating/deploy-machine-image.rst [hunk 1]
```diff
@@ -5,11 +5,14 @@
 Deploying a Machine Image
 ---------------------------------
 
-.. note:: You cannot deploy a Windows based appliance using the Deploy feature.
+.. note:: Currently, you cannot deploy a Windows based appliance using the Deploy feature.
+
+.. note:: Currently, only machine images published on Amazon, OpenStack and Microsoft Azure Resource Manager can be deployed from UForge.
```
Comment (emuus) [inline end-user/en/source/pages/templating/deploy-machine-image.rst]: I would prefer to see this grouped in 1 note 

You cannot deploy a Windows-based .... Currently, only machine images published on ..... can be deployed

**Comment by ludop-usharesoft** [comment]
I have completely merged the too notes by just adding "Linux" in the second one.
Does-it seem ok for you ?

File: end-user/en/source/pages/templating/deploy-machine-image.rst (hunk 1 above)
Comment (ludop-usharesoft) [reply end-user/en/source/pages/templating/deploy-machine-image.rst]: Sorry I think I haven't respond at the right place so I put it here too: I have completely merged the too notes by just adding "Linux" in the second one. Does-it seem ok for you ?

File: end-user/en/source/pages/templating/deploy-machine-image.rst [hunk 2]
```diff
@@ -5,9 +5,7 @@
 Deploying a Machine Image
 ---------------------------------
 
-.. note:: Currently, you cannot deploy a Windows based appliance using the Deploy feature.
-
-.. note:: Currently, only machine images published on Amazon, OpenStack and Microsoft Azure Resource Manager can be deployed from UForge.
+.. note:: Currently, only Linux machine images published on Amazon, OpenStack and Microsoft Azure Resource Manager can be deployed.
```
[the remark this answers is not in the document]
Comment (ludop-usharesoft) [reply end-user/en/source/pages/templating/deploy-machine-image.rst]: ok, done.

**Pull request merged**